### PR TITLE
[mlnx]: Add support of Mellanox SN2740 platform.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**
- Update SAI (added support of SN2740 profile).
- Update SDK to version 4.2.3130.
- Update FW to version 13.1224.0140.
- Update HW MGMT to version 1.0.0160.

### CODE REVIEW

**Comment by oleksandrivantsiv** [comment]
Retest this please

File: dockers/docker-orchagent/ipinip.json.j2 [hunk 1]
```diff
@@ -4,7 +4,7 @@
             "tunnel_type":"IPINIP",
             "src_ip":"{{ minigraph_lo_interfaces[0]['addr'] }}",
             "dst_ip":"{{ minigraph_lo_interfaces[0]['addr'] }}",
-{% if minigraph_hwsku in [ 'ACS-MSN2700', 'ACS-MSN2410', 'ACS-MSN2100' ] %}
+{% if onie_switch_asic == "mlnx" %}
```
Comment (lguohan) [inline dockers/docker-orchagent/ipinip.json.j2]: where is this var defined?

**Comment by lguohan** [comment]
retest this please

File: dockers/docker-orchagent/ipinip.json.j2 (hunk 1 above)
Comment (oleksandrivantsiv) [inline dockers/docker-orchagent/ipinip.json.j2]: sonic-cfggen reads information from ONEI:
```
root@arc-switch1026:/# sonic-cfggen --print-data
{
    "onie_arch": "x86_64",
    "onie_skip_ethmgmt_macs": "yes",
    "onie_platform": "x86_64-mlnx_msn2700-r0",
    "onie_machine_rev": "0",
    "onie_version": "2016.11-5.1.0008-9600",
    "onie_machine": "mlnx_msn2700",
    "onie_config_version": "1",
    "platform": "x86_64-mlnx_msn2700-r0",
    "onie_partition_type": "gpt",
    "onie_build_date": "\"2017-04-26T11:01+0300\"",
    "onie_switch_asic": "mlnx",
    "onie_vendor_id": "33049",
    "onie_firmware": "auto",
    "onie_kernel_version": "4.10.11"
}
```

**Pull request merged**